frontend/src/screens/researcher/studies/create/forms/participant-view.tsx

```diff
@@ -0,0 +1,332 @@
+import { EditingStudy, studySubjects, studyTopics } from '@models';
+import { Box, React, useMemo, useState, Yup } from '@common';
+import {
+    Col,
+    FieldErrorMessage,
+    FieldTitle,
+    getImageUrl,
+    InputField,
+    ResearcherButton,
+    ResearcherDetailedCheckbox,
+    SelectField,
+    StepHeader,
+    useFormContext,
+} from '@components';
+import { ImageLibrary } from '../image-library';
+import { StudyCardPreview } from '../../../../learner/card';
+import { first } from 'lodash-es';
+import { Study } from '@api';
+import { useFieldArray } from 'react-hook-form';
+import { colors } from '@theme';
+
+export const participantViewValidation = (studies: Study[], study: EditingStudy) => {
+    const allOtherStudies = useMemo(() => studies?.filter(s => 'id' in study && s.id !== study.id), [studies])
+
+    return {
+        titleForParticipants: Yup.string().when('step', {
+            is: 2,
+            then: (s: Yup.BaseSchema) =>
+                s.required('Required').max(45).test(
+                    'Unique',
+                    'This study title is already in use. Please change your study title to make it unique on Kinetic..',
+                    (value: string) => {
+                        if (!studies.length) {
+                            return true
+                        }
+                        return allOtherStudies.every(study => study.titleForParticipants?.toLowerCase() !== value?.toLowerCase())
+                    }
+                ),
+        }),
+        shortDescription: Yup.string().max(120).when('step', {
+            is: 2,
+            then: (s: Yup.BaseSchema) => s.required('Required'),
+        }),
+        longDescription: Yup.string().max(250).when('step', {
+            is: 2,
+            then: (s: Yup.BaseSchema) => s.required('Required'),
+        }),
+        studyTopic: Yup.string().when('step', {
+            is: 2,
+            then: (s: Yup.BaseSchema) => s.required('Required'),
+        }),
+        stages: Yup.array().when('step', {
+            is: 2,
+            then: Yup.array().of(
+                Yup.object({
+                    points: Yup.number().required(),
+                    feedbackTypes: Yup.array().test(
+                        'At least one',
+                        'Select at least one item',
+                        (feedbackTypes?: string[]) => (feedbackTypes?.length || 0) > 0
+                    ),
+                })
+            ),
+        }),
+        benefits: Yup.string().when('step', {
+            is: 2,
+            then: (s: Yup.BaseSchema) => s.required('Required'),
+        }),
+        imageId: Yup.string().when('step', {
+            is: 2,
+            then: (s: Yup.BaseSchema) => s.required('Required'),
+        }),
+    }
```

Comment:
Totally hear you, the nuance here is the continue/save as draft buttons are reliant on the current step's validation. ie if we are on step 1, and step 3 has a required field, it wouldn't allow us to move forward to step 2 since its technically not filled out yet